packages/database-integrations/src/database-integration-types.ts

```diff
@@ -25,6 +25,7 @@ export const databaseIntegrationTypes = [...sqlIntegrationTypes, 'mongodb'] as c
 export type DatabaseIntegrationType = (typeof databaseIntegrationTypes)[number]
 
 export const databaseIntegrationTypesWithSslSupport = [
+  'alloydb',
```

Comment:
Adding AlloyDB here because our webapp UI supports configuring SSL for this integration type.